### ISSUE TITLE
Why the switch from LRA/CuteSV to minimap2/Sniffles?

### ISSUE THREAD

**Issue by dmacguigan** [epi2me-labs/wf-human-sv#12]
### What happened?

Hello, this isn't a bug report, but I'm curious why the wf-human-sv suddenly changed from LRA+CuteSV to minimap2+sniffles2. 

### Operating System

ubuntu 18.04

### Workflow Execution

Command line

### Workflow Version

1.0.0

### Relevant log output

```shell
NA
```

**Comment by cjw85** [comment]
The primary reason to for the switch is to enable use of a single aligner across multiple workflows. We will soon be releasing a more holistic workflow that performs both SV and SNP calling (amonst other things). Computing alignments is a bottleneck so being able to switch to minimap2 for SV calling without loss of accuracy is a big plus.

**Comment by dmacguigan** [comment]
Ah I see, that explains the change in alignment software. But why the change in SV caller? Does CuteSV not perform well with minimap2 alignments as input?

**Comment by cjw85** [comment]
Correct, we have not been able to make CuteSV perform as well with minimap2 as it does with lra.

**Comment by SamStudio8** [comment]
@dmacguigan A keen eye! The supplemental information pertinent for SV calling is encoded in the primary alignment's `SA` tag. We found in our internal benchmarks that keeping the supplemental alignments themselves caused artificial coverage spikes that reduced the precision of SV calling.

**Comment by dmacguigan** [comment]
Thanks for the detailed feeback! I'm looking forward to exploring this workflow.
